### PR TITLE
Cypress tests and refactoring

### PR DESCRIPTION
**Pull request by mgavrila**

### CODE REVIEW

File: cypress/constants/enums.ts [hunk 1]
```diff
@@ -9,3 +9,15 @@ export enum ErrorMessagesEnum {
   required = 'Required'
 }
 
+export enum RoutesEnum {
+  unlock = '/unlock',
+  send = '/send',
+  dashboard = '/dashboard'
+}
+
+export enum walletIDEnum {
```
Comment (arhtudormorar) [inline cypress/constants/enums.ts:18]: WalletIDsEnum

File: cypress/e2e/BatchTransactions/helpers.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import { userData } from '../../assets/globalData';
+import { RoutesEnum } from '../../constants/enums';
+import { scSelectors } from '../SCTransactions/SCTransactionData';
+
+export const signTransactions = (selector) => {
+  let i = selector === 'swap-lock' ? 1 : 0;
```
Comment (arhtudormorar) [inline cypress/e2e/BatchTransactions/helpers.ts:6]: 'swap-lock' from constant or enum

File: cypress/support/commands.ts [hunk 3]
```diff
@@ -9,21 +9,30 @@ Cypress.Commands.add('checkUrl', (url) => {
...
-Cypress.Commands.add('login', () => {
-  cy.getSelector('keystoreBtn').click();
-  cy.checkUrl('/unlock/keystore');
-  cy.getSelector('submitButton').click();
-  cy.checkUrl('/unlock/keystore');
+Cypress.Commands.add('login', (walletID, selector) => {
+  cy.session(walletID, () => {
+    cy.visit('/');
+    cy.contains(selector).click();
+    if (selector === 'Connect') {
```
Comment (arhtudormorar) [inline cypress/support/commands.ts:16]: 'Connect' from const or enum everywhere

File: src/pages/Dashboard/widgets/BatchTransactions/BatchTransactions.tsx [hunk 4]
```diff
@@ -133,13 +133,15 @@ export const BatchTransactions = () => {
     <div className='flex flex-col gap-6'>
       <div className='flex gap-2 items-center'>
         <Button
+          data-testid='sign-auto-send'
```
Comment (arhtudormorar) [inline src/pages/Dashboard/widgets/BatchTransactions/BatchTransactions.tsx:136]: use DataTestIDsEnum

**Review by CiprianDraghici — APPROVED** [review]

**Pull request merged**